src/bandersnatch/verify.py

```diff
@@ -90,14 +91,14 @@ async def delete_unowned_files(
 
 
 async def verify(
-    config,
-    json_file,
-    mirror_base_path,
-    all_package_files,
-    args,
-    executor,
-    releases_key="releases",
-):
+    config: ConfigParser,
+    json_file: str,
+    mirror_base_path: Path,
+    all_package_files: List,
```

Comment:
List can take the type in the list - List[Path]
- The more info you give typing the more it will help us spot bugs etc. :)

Reply:
Yup, thanks, I was a little confused on what the sub-type was!